mobile/native/src/trade/order/handler.rs

```diff
@@ -90,34 +89,28 @@ pub async fn submit_order(
 
     db::insert_order(order.clone()).map_err(SubmitOrderError::Storage)?;
 
-    let runtime = get_or_create_tokio_runtime().expect("to be ablet to get runtime");
-    if let Err(err) = runtime
-        .spawn_blocking({
```

Comment:
📣 We should not block and async task. Removing the `spawn_blocking` is not a good idea.

Reply:
> 📣 We should not block and async task. Removing the `spawn_blocking` is not a good idea.

Can you explain that? It looks weird to me because at the end we are still awaiting the task.

I've removed it because we are not doing it when calling the trade api and I haven't seen any regressions during testing.

The fix should be possible without removing the spawn_blocking. The important part is to not call the post order api before the order has been committed.

Reply:
> Can you explain that? It looks weird to me because at the end we are still awaiting the task.

You can wait for the task to complete, but the important thing is that with `spawn_blocking` the task will be run in a _blocking_ thread pool where blocking is safe.

Reply:
Yep, we have an issue with this kind of thing in several places.